devices.js

```diff
@@ -12940,6 +12940,40 @@ const devices = [
         description: 'AOne 250W smart rotary dimmer module',
         extend: generic.light_onoff_brightness,
     },
+    {
+        zigbeeModel: ['SmartPlug51AU'],
+        model: 'SmartPlug51AU',
+        vendor: 'Aurora Lighting',
+        description: 'Aurora Smart plug',
+        supports: 'on/off, power measurement, temperature',
+        fromZigbee: [fz.identify, fz.on_off, fz.electrical_measurement_power, fz.metering_power, fz.device_temperature, fz.occupancy],
+        toZigbee: [tz.on_off],
+        meta: {configureKey: 6, multiEndpoint: true},
+        endpoint: (device) => {
+            return {
+                default: 2,
```

Comment:
- Can you provide a link to this device?
- Does this device indeed support multiple endpoints? Because in your HA discovery you didn't support this (https://github.com/Koenkk/zigbee2mqtt/pull/4562/files#diff-0c056233b68fd3653afb296f98947c8fR1843). This only make sense to add when there are e.g. multiple gangs to control